DigitalAIOfficial/Workflows/ArgoCD/Workflow_ArgoCD setup application.yaml

```diff
@@ -40,6 +70,7 @@ spec:
               - remote
             gitRepo: "${gitRepo}"
             path: "${path}"
+            chart: "${chart}"
```

Comment:
xl apply is failing because of chart variable. please check it out, probably some default value...

Reply:
works on my machine :D
jk, did you try using the right argo plugin version? (from branch D-37308)